src/components/tutorial.jsx

```diff
@@ -0,0 +1,87 @@
+import React        from 'react';
+import Modal        from 'react-modal';
+import { connect }  from '../stores/connect';
+import                   '../assets/sass/_tutorial.scss';
+
+const customStyles = {
+    content: {
+        top        : '50%',
+        left       : '50%',
+        right      : 'auto',
+        bottom     : 'auto',
+        marginRight: '-50%',
+        transform  : 'translate(-50%, -50%)',
+    },
+};
+
+Modal.setAppElement('#root');
+
+const Tutorial = ({
+    setShowAgain,
+    modalOpen,
+    strategy,
+    setStrategy,
+    tradeOptions,
+    tradeOptionValue,
+    closeModal,
+    setTradeOption,
+    handleSubmit,
+}) => {
+    const { market, contract, stake, size, loss, profit } = tradeOptions;
+
+    return (
+        <Modal
+            isOpen={modalOpen}
+            contentLabel='Example Modal'
+            shouldCloseOnOverlayClick={true}
+            onRequestClose={closeModal}
+            style={customStyles}
```

Comment:
Could we define the styling rules its own separate sass file rather than inlining them?

Reply:
it's just a temporally component, is it necessary?